test/test_series.py

```diff
@@ -64,19 +52,28 @@ def test_time_broadcasting_data_single_shard():
         ),
         end='',
     )
-    nodes[broadcasting_node].broadcast_collation(0, num_collations, collation_size, collation_time)  # 1MB
+    nodes[broadcasting_node].broadcast_collation(0, num_collations, collation_size, collation_time)
     print("done")
```

Comment:
Sorry I failed to raise the question in the first pull request but do we need to wait a few seconds for the shard subscription messages and new collation message to reach every nodes?

Reply:
Yes, we need. `get_log_time` is already blocking wait for the logs